cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -281,6 +281,7 @@ func newControllerInitializers() map[string]InitFunc {
 	controllers["statefuleset"] = startStatefulSetController
 	controllers["cronjob"] = startCronJobController
 	controllers["certificatesigningrequests"] = startCSRController
+	controllers["ttl"] = startTTLController
```

Comment:
I'm fine with changing but i couldn't come up with any better name - so if you could suggest something better, I'm happy to change that.